src/components/Chart/Chart.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Line } from "react-chartjs-2";
+
+export default function Chart(props) {
+  const yAxisMax = Math.max(...props.data);
+  const yAxisMin = Math.min(...props.data);
+
+  const data = {
+    labels: [...Array(props.data.length).keys()],
```

Comment:
can you explain this?

Reply:
That's just to have an array with the integers from 0 to `data.length-1`